### PR TITLE
Fix GitHub labels used in our weekly releases

### PR DESCRIPTION
**Pull request by emilk**
See https://github.com/rerun-io/rerun/pull/3057:

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3061) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3061)
- [Docs preview](https://rerun.io/preview/pr%3Aemilk%2Ffix-weekly-release-labels/docs)
- [Examples preview](https://rerun.io/preview/pr%3Aemilk%2Ffix-weekly-release-labels/examples)
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: .github/workflows/weekly_release.yml [hunk 1]
```diff
@@ -40,9 +40,7 @@ jobs:
           branch: "weekly-release-${{ env.CRATE_VERSION }}"
           commit-message: "[weekly-release] ${{ env.CRATE_VERSION }}"
           title: "Release ${{ env.CRATE_VERSION }}"
-          labels: |
-            "⛴ release"
-            "exclude from changelog"
+          labels: "⛴ release, exclude from changelog"
```
Comment (jprochazk) [inline .github/workflows/weekly_release.yml:43]:
```suggestion
          labels: |
            ⛴ release
            exclude from changelog
```
YAML moment, I think this should work if we just remove the quotes

Comment (jprochazk) [reply .github/workflows/weekly_release.yml:43]: Just guessing based on what the output of the action was in the previous run https://github.com/rerun-io/rerun/actions/runs/5927259856/job/16070405253#step:6:7

Comment (emilk) [reply .github/workflows/weekly_release.yml:43]: god damn I hate yaml

Comment (emilk) [reply .github/workflows/weekly_release.yml:43]: But yeah, I think you are right: https://github.com/peter-evans/create-pull-request#reference-example

**Review by jprochazk — APPROVED** [review]

**Pull request merged**